content/en/02-installing-pixie/01-requirements.md

```diff
@@ -69,23 +62,42 @@ Pixie runs on Linux nodes only.
 | Linux   | Supported       | v4.14+            |
 | Windows | Not Supported   | Not in roadmap    |
 
-### Linux Distributions
+### Linux Distribution
 
-The following is a list of Linux distributions that have been tested.
+The following table lists Linux distributions that are known to work with Pixie.
 
 |              |  Version              |
 |:-----------  |  :------------------- |
-| Ubuntu       |  18.04+               |
+| CentOS       |  7.3+                 |
 | Debian       |  10+                  |
-| RHEL         |  8+                   |
-| CentOS       |  7.3+                  |
-
-Pixie may also work on other distributions.
+| RedHat Enterprise Linux |  8+        |
+| Ubuntu       |  18.04+               |
 
 <Alert variant="outlined" severity="info">We recommended that Linux headers be installed on a cluster's nodes.</Alert>
+<Alert variant="outlined" severity="info">Pixie may also work on other distributions. If you find an unlisted Linux distribution that works well, please let us know by opening a GitHub issue or by creating a pull request that updates <a href="https://github.com/pixie-io/docs.px.dev/blob/main/content/en/02-installing-pixie/01-requirements.md">this doc</a>.</Alert>
+
+## CPU
+
+Pixie requires an `x86-64` architecture.
+
+|         | Support           |
+| :------ | :---------------- |
+| x86-64  | Supported         |
+| ARM     | Not Supported     |
 
 ## Network Traffic
 
 Pixie's [Vizier](/reference/architecture/#vizier) module sends outgoing HTTPS/2 requests to Pixie's [Cloud](/reference/architecture/#cloud) on port `443`.
 
 Your cluster's telemetry data flows through Pixie's Cloud via a reverse proxy as encrypted traffic without any persistence. This allows users to access data without being in the same VPC/network as the cluster. Pixie offers [end-to-end encryption](/about-pixie/faq#data-collection-how-does-pixie-secure-its-data) for telemetry data in flight.
+re
+
+## Memory
+
+Memory requirements for your cluster nodes are as follows:
+
+| Minimum   | Notes                                                                  |
+| :-------- | :--------------------------------------------------------------------- |
+| 1GiB      | To accommodate application pods, 4GiB+ total per node is recommended.  |
```

Comment:
People get confused about total memory vs Pixie memory. I think it's fairly clear but let's be even more clear. 

To accommodate application pods, we recommend using no more than 25% of the nodes' total memory for Pixie.